appConfig/appConfig.yaml

```diff
@@ -44,3 +44,14 @@
   type: "keyvault"
 - key: TP_QUEUE_ADDRESS
   value: "defra.trade.plp.parsed"
+- key: FORM_RECOGNIZER_ENDPOINT
+  value: "https://adpaipocuksdi.cognitiveservices.azure.com/"
```

Comment:
Would this value not be different for preprod and prod?

Reply:
good point will change that